pkg/services/ngalert/notifier/channels/dingding.go

```diff
@@ -92,14 +92,13 @@ func (dd *DingDingNotifier) Notify(ctx context.Context, as ...*types.Alert) (boo
 	// Refer: https://open-doc.dingtalk.com/docs/doc.htm?treeId=385&articleId=104972&docType=1#s9
 	messageURL := "dingtalk://dingtalkclient/page/link?" + q.Encode()
 
-	var tmplErr error
-	tmpl, _ := TmplText(ctx, dd.tmpl, as, dd.log, &tmplErr)
+	expand, _ := TmplText(ctx, dd.tmpl, as, dd.log)
```

Comment:
what do you think about the name `expandFunc` instead? I think it makes it more obvious what it is